.vscode/settings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "editor.formatOnSave": true,
```

Comment:
omit this feature as it is a personal preference, for example, I like to format whenever I am ready to format + we format on commit as a second safety net.